dataland-frontend/src/assets/css/main.css

```diff
@@ -0,0 +1,8 @@
+#app {
```

Comment:
Is this styling needed at this point? Materialize css should style properly at the current stage.

Reply:
testet, is needed